modules/internal/ChapelArray.chpl

```diff
@@ -3448,10 +3448,16 @@ module ChapelArray {
       return this(this.domain.high);
     }
 
+    /* This function is deprecated use .last instead */
```

Comment:
Note that with the new support for the deprecated keyword generating deprecation messages in the documentation, this documentation comment is no longer necessary.  You don't have to remove it, though, or you could go so far as to explicitly link to `last` (see [the inline markup section](https://chapel-lang.org/docs/latest/tools/chpldoc/chpldoc.html#inline-markup-2) of the chpldoc documentation for how to go about doing that) or put the comment in a `.. warning::` block (which follows the [paragraph markup syntax](https://chapel-lang.org/docs/latest/tools/chpldoc/chpldoc.html#paragraph-level-markup)) so that it pops more.

Reply:
Very cool, I now see why the deprecated keyword is so nice! I’ll remove the comment on the principle of trying to avoid redundant information.

I just tried cross referencing the function in the deprecation comment to and it looks like Sphinx will generate the link in the comment the way we want. Unfortunately, if I do that it will print the message out to the console literally (including the Sphinx markup). Would you be opposed to having the Sphinx markup filtered out before printing the deprecation comment to the terminal?  I’ve added a PR to do this: https://github.com/chapel-lang/chapel/pull/18540

Reply:
I was wondering about that exact tension recently and like the proposal to make this easy.

Reply:
Looks good, though I think you'll need to merge #18540 first and rebase to ensure the tests still pass